doc/man3/PKCS12_item_decrypt_d2i.pod

```diff
@@ -39,6 +39,9 @@ PKCS12_item_i2d_encrypt() and PKCS12_item_i2d_encrypt_ex() encrypt an ASN.1
 object I<it> using the algorithm I<algor> and password I<pass> of length
 I<passlen>, returning an encoded object in I<obj>.
 
+If I<zbuf> is set then for encrypt the buffer containing the input encoding is
```

Comment:
set to 1? (or is non zero)

Maybe move this up into 2 separate places above... One in the PKCS12_item_decrypt_d2i() section.. and one in the decrypt. That should make it clearer.